gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -746,13 +746,25 @@ end
 
 -- Simple client-based idle checking
 local idle = {
-	ang = nil,
-	pos = nil,
+	btn = 0,
 	mx = 0,
 	my = 0,
-	t = 0
+	t = 0,
 }
 
+---
+-- Sniff for any button inputs to prevent scamming the AFK timer.
+-- @realm client
+hook.Add("SetupMove", "DeathrunIdleCheck", function(_, mv)
```

Comment:
```suggestion
hook.Add("SetupMove", "TTT2IdleCheck", function(_, mv)
```

Reply:
i've been awake too long, i got the angles to stop interfering and suddenly everything else slips right past